alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/ClientCertificateModel.java

```diff
@@ -10,6 +11,7 @@ public class ClientCertificateModel extends AlertSerializableModel implements Ob
     private static final long serialVersionUID = -2254327445871324025L;
     private String keyPassword;
     private String keyContent;
+    @JsonProperty("clientCertificateContent")
     private String certificateContent;
```

Comment:
We don't need to add the JsonProperty annotation here since we're creating this as a net new property. It would likely be easier to just rename the field from "certificateContent" to "clientCertificateContent" and refactor appropriately

Reply:
This would be a one line change here vs refactoring multiple files. We have examples of different property/alias name in both alert and hub. Open to the latter however.

Reply:
This is a new feature not an existing feature.  I would change the field name to clientCertificateContent.  Yes, multiple files will change but that's ok. 7 files would change. The feature this is related to is the only feature for the release.  I would like the changes made as complete as possible before release.  If this was released code then yes I would agree with the annotation 1 line fix.  This isn't released and we would like the variable changed. We try to avoid adding the annotations unless we absolutely need to as is the case with released features where we need to maintain backwards compatibility.

Reply:
Cool, just needed context on why some places we are ok with different property name but not in this instance. Renamed to clientCertificateContent